### ISSUE TITLE
Korean UA in CP extremely overpowered

### ISSUE THREAD

**Issue by nOxrax** [LoneGazebo/Community-Patch-DLL#1762]
1. Mod Version
4/23b beta with dll hotfix

2. Mod List
CP
Custom Advanced Setup Screen (+ playing hotseat)

3. Type of error
CBP code affecting CP (I think).

4. Description
The Korean UA in CP is to get a tech boost each time a scientific building is built in the capital. It used to be a modest boost, but now it is enormous! My guess is that the tech boost is balanced for CBP, where the general science values are higher.

**Comment by LoneGazebo** [comment]
Are you playing a marathon game? It scales by gamespeed now. That was a bug from Firaxis that I fixed.

**Comment by nOxrax** [comment]
Yes, I play marathon so this bugfix would explain the sudden increase. But still, the Korean UA is off scales! For reference, I gained about 80 000 tech points when I built the Research Lab, and I advanced three techs! Add to this all bonuses I got from building the university and oxford university and public school, each of them advancing me several techs. This is an extreme UA and I doubt it can be WAD. Are you sure the values are right?
